turborepo-tests/helpers/setup_example_test.sh

```diff
@@ -1,37 +1,77 @@
 #!/usr/bin/env bash
 
-set -eo pipefail
```

Comment:
Probably should bring this back. If something fails in this script, actual test execution will probably also fail, so it would be good to just exit early.